packages/rs-dpp/src/state_transition/mod.rs

```diff
@@ -209,25 +223,25 @@ macro_rules! call_errorable_method_identity_signed {
         }
     };
 }
-
-macro_rules! call_static_method {
-    ($state_transition:expr, $method:ident ) => {
-        match $state_transition {
-            StateTransition::DataContractCreate(_) => DataContractCreateTransition::$method(),
-            StateTransition::DataContractUpdate(_) => DataContractUpdateTransition::$method(),
-            StateTransition::DocumentsBatch(_) => DocumentsBatchTransition::$method(),
-            StateTransition::IdentityCreate(_) => IdentityCreateTransition::$method(),
-            StateTransition::IdentityTopUp(_) => IdentityTopUpTransition::$method(),
-            StateTransition::IdentityCreditWithdrawal(_) => {
-                IdentityCreditWithdrawalTransition::$method()
-            }
-            StateTransition::IdentityUpdate(_) => IdentityUpdateTransition::$method(),
-            StateTransition::IdentityCreditTransfer(_) => {
-                IdentityCreditTransferTransition::$method()
-            }
-        }
-    };
-}
+// TODO unused macros below
```

Comment:
Then remove or mark as dead code

Reply:
can't mark macro as dead it seems; if @QuantumExplorer agrees to remove I will

Reply:
make public instead

Reply:
what do you mean?

Reply:
make the macro public

Reply:
We agreed to leave commented for now.